text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,376 @@ nit-picking the "gotchas" in code examples. :heart:
 
 # Summary
 
-Brief explanation of the feature.
+Migrate Headless-styles to use [Panda CSS](https://panda-css.com/) under the hood via
+a new public Pando preset configuration option called `@pluralsight/panda-preset`.
+
+This will allow both Headless-styles and the React lib to be more flexible and
+configurable for future use-cases by leveraging the power of Panda CSS. Additionally,
+it will increase the overall performance of the React lib by reducing the amount of
+CSS that is shipped to the client (via Atomic-CSS).
 
 # Basic example
 
-If the proposal involves a new or changed API, include a basic code example.
-Omit this section if it's not applicable.
+This is a basic example of using a new Pando CLI tool to install and setup the
+local project with all dependencies and configuration needed to use Pando with
+Panda CSS (I know that's a lot on the brain to read - I think they are trolling
+us with the name :stuck_out_tongue_winking_eye: ).
+
+```bash
+$ pnpx pando setup --typescript
+
+📥 Installing Pando dependencies
+
+- @pluralsight/headless-styles
+- @pluralsight/icons
+- @pluralsight/react
+
+📥 Installing Panda CSS dependencies (dev)
+
+- @pandacss/dev
```

Comment:
What's the relationship between dependencies, is `pandacss/dev` `postcss` and `@pluralsight/panda-preset` peer dependencies of `headless-styles`?

Is there a reason to not make those direct dependencies to simplify implementation of Pando?
i.e. for consumers to just install and understand the `@pluralsight` packages and not worry about what's underneath.

Are versions of those dependencies coupled to the `headless-styles` versions? So if I find a compatibility issue or vulnerability with the panda or postcss version, can I update them myself or will I be blocked by a supported version range in Pando?

If they are coupled, does it make a case for them to be direct dependencies so management of those compatibility/vulnerability issues can be centralised on Pando?

Reply:
Yeah I can see postcss make total sense as a peer dep since it's quite likely it's already a dependency for most projects. But I agree I think all the panda related stuff would probably be totally fine and even preferred as direct dependencies.

Reply:
@timkinnane @josh-degraw great question and yes, essentially, the CLI would take care of everything and the DX should no longer have to worry about multiple packages or "maintaining" so to speak.

So think something like more of the CLI will manage Pando and Panda-CSS/Post-CSS will manage styling. So, theoretically, HS will not exist in the future and the "core" packages should never/rarely be used. Instead it's basically the CLI probably just installing:

```bash
post-css
@panda-css
@pluralsight/react
@pluralsight/pando-preset
```

...then setting everything up for your local env. No work should be required for users in v1.